### PR TITLE
fix: Bump fluence-js to 0.27.3 and pin it

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,67 +103,8 @@ Pull request and release process:
 # Commands
```
Comment (shamsartem) [inline README.md:103]: In order for this file to be generated correctly - please follow the guide in README for Contributing

Comment (shamsartem) [reply README.md:103]: This file is generated after commit. Maybe I should review this process so this kind of PR's would not require building the project

File: package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "clean": "shx rm -rf .fluence && shx rm -rf fluence.yaml && shx rm -rf src/aqua && shx rm -rf src/ts  && shx rm -rf src/js"
   },
   "dependencies": {
-    "@fluencelabs/fluence": "^0.24.1",
+    "@fluencelabs/fluence": "0.27.3",
```
Comment (shamsartem) [inline package.json:37]: Please also update `AQUA_RECOMMENDED_VERSION` at `src/lib/const.ts`

**Review by shamsartem — CHANGES_REQUESTED** [review]
Thanks for the update! Please fix these small issues

File: README.md [hunk 3]
```diff
@@ -196,7 +196,7 @@ _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomp
 
 ## `fluence build`
 
-Build all application services, described in fluence.yaml
+Build all application services, described in [33mfluence.yaml[39m
```
Comment (nahsi) [inline README.md:199]:
```suggestion
Build all application services, described in fluence.yaml
```

File: README.md [hunk 4]
```diff
@@ -244,7 +244,7 @@ EXAMPLES
 
 ## `fluence dep i`
 
-Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
+Install all project dependencies (dependencies are cached inside [33m.fluence[39m directory of the current user)
```
Comment (nahsi) [inline README.md:247]:
```suggestion
Install all project dependencies (dependencies are cached inside fluence directory of the current user)
```

File: README.md [hunk 5]
```diff
@@ -353,7 +353,7 @@ EXAMPLES
 
 ## `fluence dependency i`
 
-Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
+Install all project dependencies (dependencies are cached inside [33m.fluence[39m directory of the current user)
```
Comment (nahsi) [inline README.md:356]:
```suggestion
Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
```

File: README.md [hunk 6]
```diff
@@ -379,7 +379,7 @@ EXAMPLES
 
 ## `fluence dependency install`
 
-Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
+Install all project dependencies (dependencies are cached inside [33m.fluence[39m directory of the current user)
```
Comment (nahsi) [inline README.md:382]:
```suggestion
Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
```

File: README.md [hunk 7]
```diff
@@ -459,7 +459,7 @@ EXAMPLES
 
 ## `fluence deploy`
 
-Deploy application, described in fluence.yaml
+Deploy application, described in [33mfluence.yaml[39m
```
Comment (nahsi) [inline README.md:462]:
```suggestion
Deploy application, described in fluence.yaml
```

File: README.md [hunk 8]
```diff
@@ -593,7 +593,7 @@ EXAMPLES
 
 ## `fluence module add [PATH | URL]`
 
-Add module to service.yaml
+Add module to [33mservice.yaml[39m
```
Comment (nahsi) [inline README.md:596]:
```suggestion
Add module to service.yaml
```

File: README.md [hunk 9]
```diff
@@ -637,7 +637,7 @@ EXAMPLES
 
 ## `fluence module remove [NAME | PATH | URL]`
 
-Remove module from service.yaml
+Remove module from [33mservice.yaml[39m
```
Comment (nahsi) [inline README.md:640]:
```suggestion
Remove module from service.yaml
```

File: README.md [hunk 10]
```diff
@@ -719,7 +719,7 @@ _See code: [dist/commands/run.ts](https://github.com/fluencelabs/fluence-cli/blo
 
 ## `fluence service add [PATH | URL]`
 
-Add service to fluence.yaml
+Add service to [33mfluence.yaml[39m
```
Comment (nahsi) [inline README.md:722]:
```suggestion
Add service to fluence.yaml
```

File: README.md [hunk 11]
```diff
@@ -765,7 +765,7 @@ EXAMPLES
 
 ## `fluence service remove [NAME | PATH | URL]`
 
-Remove service from fluence.yaml
+Remove service from [33mfluence.yaml[39m
```
Comment (nahsi) [inline README.md:768]:
```suggestion
Remove service from fluence.yaml
```

File: README.md [hunk 12]
```diff
@@ -244,7 +244,7 @@ EXAMPLES
 
 ## `fluence dep i`
 
-Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
+Install all project dependencies (dependencies are cached inside fluence directory of the current user)
```
Comment (nahsi) [inline README.md:247]:
```suggestion
Install all project dependencies (dependencies are cached inside .fluence directory of the current user)
```

**Review by shamsartem — APPROVED** [review]
Thanks!

**Pull request merged**